packages/ui/src/Table.tsx

```diff
@@ -311,16 +341,18 @@ export const Column = styled.td<{
   width?: number | string
   alignText?: string
   withTitle?: boolean
-  isTitleAString?: boolean
```

Comment:
if we are not using isTitleAString and withTitle. we need to remove this usage fully.

Reply:
We are using isTitleAString, but withTitle can be deleted